### PR TITLE
[TRANSLATION] Update Polish translation

### PR DESCRIPTION
**Pull request by m4sk1n**

### CODE REVIEW

File: reactos/base/applications/clipbrd/lang/pl-PL.rc [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/*
```
Comment (ThFabba) [inline reactos/base/applications/clipbrd/lang/pl-PL.rc]: Adding this file doesn't do anything, you need to also reference it in clipbrd.rc

File: reactos/base/setup/reactos/lang/pl-PL.rc [hunk 2]
```diff
@@ -6,6 +6,7 @@
  *    Update - niski (bjauy@o2.pl)
  *    Updated by Caemyr (Olaf Siejka), Mar 2011
  *    UTF-8 conversion by Caemyr (May, 2011)
+ *    Updated by Marcin Mikołajczak <me@m4sk.in> (6-06-2017)
```
Comment (ThFabba) [inline reactos/base/setup/reactos/lang/pl-PL.rc]: I'd recommend a less ambiguous date format.

File: reactos/base/system/services/lang/pl-PL.rc [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/*
```
Comment (ThFabba) [inline reactos/base/system/services/lang/pl-PL.rc]: Needs to be referenced in the main .rc. Please test your changes -- you'd notice that this is missing.

**Comment by m4sk1n** [comment]
@ThFabba Now should be ok

File: reactos/base/system/userinit/lang/pl-PL.rc [hunk 4]
```diff
@@ -13,7 +14,7 @@ CAPTION "ReactOS LiveCD"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "IDB_LOGO", IDC_LOCALELOGO, "Static", WS_CHILD | WS_VISIBLE | SS_OWNERDRAW, 18, 0, 290, 99
-    RTEXT "Language:", IDC_STATIC, 20, 109, 106, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
+    RTEXT "Język:", IDC_STATIC, 20, 109, 106, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_LANGUAGELIST, 132, 107, 176, 142, CBS_DROPDOWNLIST | CBS_SORT | WS_VSCROLL | WS_CHILD | WS_VISIBLE | WS_TABSTOP
     RTEXT "Keyboard layout:", IDC_STATIC, 20, 132, 106, 11, WS_CHILD | WS_VISIBLE | WS_GROUP
```
Comment (ThFabba) [inline reactos/base/system/userinit/lang/pl-PL.rc]: Didn't want to translate this one? ;)

Comment (m4sk1n) [inline reactos/base/system/userinit/lang/pl-PL.rc]: sorry for that

**Comment by m4sk1n** [comment]
@ThFabba, can you review it now?

**Comment by m4sk1n** [comment]
@ThFabba

**Comment by Saibamen** [comment]
Looks ok for me

**Pull request closed without merging**